index.d.ts

```diff
@@ -1,11 +1,11 @@
 /** 
  * Typings for twitter-lite 
  * 
- * @version 0.9.4
+ * @version 0.10-1.0
  * @author Floris de Bijl <@fdebijl>
  * 
  * @example
- * import { Twitter } from 'twitter-lite';
+ * import Twitter = require('twitter-lite');
```

Comment:
imo this example should either just be `import Twitter from 'twitter-lite'` or show both as an example.
afaik the difference is only the version of module importing (ES5 vs ES6), and ES6 should be the recommended, especially since TS works that way.

Also, if the change i propose in my second comment was applied, [the docs](https://www.typescriptlang.org/docs/handbook/modules.html#export-all-as-x) say that `import Twitter = require('twitter-lite');` would be needed to be changed to the standard ES5 syntax `const Twitter = require('twitter-lite')`